aospy/calc.py

```diff
@@ -10,14 +10,18 @@
 import numpy as np
 import xarray as xr
 
-from .constants import Constant, grav
 from . import internal_names
 from . import utils
 from .var import Var
 
 
 logging.basicConfig(level=logging.INFO)
 
+GRAV = (
```

Comment:
@micahkim23 @spencerahill would we like to make these DataArrays rather than tuples (as suggested in https://github.com/spencerahill/aospy/issues/50#issuecomment-299206776), or is it not worth it?  This would give the benefit of named attributes, and remove the need to select the first element when doing mathematical operations.

Reply:
@spencerkclark I think the DataArrays are better than tuples. Later on, having tuples means that we will have to keep track of which indices we're selecting from the tuples which is cumbersome.

Reply:
+1 on DataArrays.  I had been thinking just floats but DataArrays much better.